sycl/include/CL/sycl/backend.hpp

```diff
@@ -213,6 +217,41 @@ make_buffer(const typename backend_traits<Backend>::template input_type<
       AvailableEvent);
 }
 
+template <backend Backend, typename T, int Dimensions = 1,
+          typename AllocatorT = buffer_allocator>
+typename std::enable_if<Backend == backend::ext_oneapi_level_zero,
+                        buffer<T, Dimensions, AllocatorT>>::type
+make_buffer(
+    const backend_input_t<backend::ext_oneapi_level_zero,
+                          buffer<T, Dimensions, AllocatorT>> &BackendObject,
+    const context &TargetContext, event AvailableEvent) {
+  detail::pi::PiMem PiBuffer = detail::make_pi_mem(
+      detail::pi::cast<pi_native_handle>(BackendObject.NativeHandle),
+      TargetContext,
+      BackendObject.Ownership == ext::oneapi::level_zero::ownership::keep,
+      Backend);
+  return detail::make_buffer_helper<T, Dimensions, AllocatorT>(
+      detail::pi::cast<pi_native_handle>(PiBuffer), TargetContext,
+      AvailableEvent);
+}
+
+template <backend Backend, typename T, int Dimensions = 1,
+          typename AllocatorT = buffer_allocator>
+typename std::enable_if<Backend == backend::ext_oneapi_level_zero,
+                        buffer<T, Dimensions, AllocatorT>>::type
+make_buffer(
+    const backend_input_t<backend::ext_oneapi_level_zero,
+                          buffer<T, Dimensions, AllocatorT>> &BackendObject,
+    const context &TargetContext) {
+  detail::pi::PiMem PiBuffer = detail::make_pi_mem(
+      detail::pi::cast<pi_native_handle>(BackendObject.NativeHandle),
+      TargetContext,
+      BackendObject.Ownership == ext::oneapi::level_zero::ownership::keep,
+      Backend);
+  return detail::make_buffer_helper<T, Dimensions, AllocatorT>(
+      detail::pi::cast<pi_native_handle>(PiBuffer), TargetContext);
+}
+
```

Comment:
I think these specializations should be moved to https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/ext/oneapi/backend/level_zero.hpp

Reply:
Moved, thanks.